client/src/components/Workflow/Editor/Tools/ToolBar.vue

```diff
@@ -316,6 +356,45 @@ const toggleVisibilityButtonTitle = computed(() => {
                     Remove all
                 </BButton>
             </div>
+
+            <div v-if="currentTool === 'boxSelect'" class="option buttons">
+                <BButtonGroup>
+                    <BButton
+                        :pressed="toolbarStore.boxSelectMode === 'add'"
+                        class="button"
+                        data-option="select-mode-add"
+                        variant="outline-primary"
+                        title="add items to selection"
+                        @click="toolbarStore.boxSelectMode = 'add'">
+                        Add to selection
+                    </BButton>
+                    <BButton
+                        :pressed="toolbarStore.boxSelectMode === 'remove'"
+                        class="button"
+                        data-option="select-mode-remove"
+                        variant="outline-primary"
+                        title="remove items from selection"
+                        @click="toolbarStore.boxSelectMode = 'remove'">
+                        Remove from selection
+                    </BButton>
+                </BButtonGroup>
+            </div>
+        </div>
+
+        <div v-if="anySelected" class="selection-options">
+            <span>{{ selectedCountText }}</span>
+
+            <BButtonGroup>
+                <BButton class="button" title="clear selection" @click="deselectAll">
+                    Clear <FontAwesomeIcon icon="fa-times" />
+                </BButton>
+                <BButton class="button" title="duplicate selected" @click="duplicateSelection">
+                    Duplicate <FontAwesomeIcon icon="fa-clone" />
+                </BButton>
+                <BButton class="button" title="delete selected" @click="deleteSelection">
```

Comment:
```suggestion
                <BButton class="button" title="send to subworkflow" @click="duplicateSelectionToSubworkflow">
                    Duplicate to Subworkflow<FontAwesomeIcon icon="fa-clone" />
                </BButton>
                <BButton class="button" title="delete selected" @click="deleteSelection">
```
😆 of course not in this PR, but this is really so cool to see!